cron.yaml

```diff
@@ -2,6 +2,9 @@ cron:
 - description: daily email about mapreduce job statuses
   url: /cron/mail/admin/job_status
   schedule: every day 16:00
+- description: weekly copy of a user's dashboard statistics
```

Comment:
I think this was left in by mistake, sorry. Please remove? It's duplicated below.